install/kots/manifests/gitpod-registry-s3-backend.yaml

```
@@ -11,5 +11,5 @@ metadata:
   annotations:
     kots.io/when: '{{repl and (ConfigOptionEquals "reg_incluster" "0") (ConfigOptionEquals "reg_s3storage" "1") }}'
```

The `kots.io/when` annotation needs updating too

```suggestion
    kots.io/when: '{{repl and (ConfigOptionEquals "reg_incluster" "0") (ConfigOptionEquals "reg_incluster_storage" "s3") }}'
```